spec/brcobranca/boleto/itau_spec.rb

```diff
@@ -197,10 +197,8 @@
     expect(boleto_novo.nosso_numero_dv).to eql(7)
   end
 
-  it 'Busca logotipo do banco' do
-    boleto_novo = described_class.new
-    expect(File.exist?(boleto_novo.logotipo)).to be_truthy
-    expect(File.stat(boleto_novo.logotipo).zero?).to be_falsey
+  describe 'Busca logotipo do banco' do
+    it_behaves_like 'busca_logotipo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.